### ISSUE TITLE
Reading 'foo'.length fails

### ISSUE THREAD

**Issue by sholladay** [dustinspecker/eslint-plugin-no-use-extend-native#79]
Code that reads from a string literal's `length` property triggers errors from this plugin.

___

```sh
$ npm install -g eslint eslint-plugin-no-use-extend-native
$ eslint --version
3.2.2
```

This code *should* pass, but does not, regardless of wrapping it in parentheses, etc.

```sh
$ echo "console.log('foo'.length)" | eslint --stdin --rule='no-use-extend-native/no-use-extend-native:2' --plugin='no-use-extend-native'

<text>
  1:13  error  Avoid using extended native objects  no-use-extend-native/no-use-extend-native

✖ 1 problem (1 error, 0 warnings)
```

This code should also pass and does, successfully.

```sh
$ echo "var x = 'foo'; console.log(x.length)" | eslint --stdin --rule='no-use-extend-native/no-use-extend-native:2' --plugin='no-use-extend-native'
$ 
```

**Comment by dustinspecker** [comment]
Thanks for reporting this.

Interesting that

```js
var x = 'foo'.length;
```

works, but

```js
console.log('foo'.length)
```

does not.

I'll try to take a look at this tonight after work.

**Comment by dustinspecker** [comment]
Ok, looked at it this morning. Should be released soon. Just waiting for Travis to run tests.
